README.md

```diff
@@ -79,7 +79,7 @@ would change game rate to 10 frames per second:
 
 ## Developing
 
-Before submitting patches, please run the tests:
+Before submitting the patches, please run the tests:
```

Comment:
I don't think this is needed, but I'm not native English speaker. Can you explain why this needed?